stylesheets/commons/Brackets.less

```diff
@@ -587,6 +587,10 @@ div.brkts-popup-body-element-thumbs {
 	display: inline-block;
 }
 
+.brkts-popup-body-element > .brkts-popup-body-game {
+	display: flex;
+}
```

Comment:
should i make this sc(2)/wc/sg spwcific?

Reply:
tests via browser dev tools on R6, dota2, fifa and rl
could not see any impact in my tests
(obviously on sc2 it has the intended impact)